sos/report/plugins/pulp.py

```diff
@@ -128,6 +130,11 @@ def setup(self):
             "qpid-stat -%s --ssl-certificate=%s -b amqps://localhost:5671" %
             (opt, self.messaging_cert_file) for opt in "quc"
         ])
+        self.add_cmd_output(
+            "sudo -u pulp PULP_SETTINGS='/etc/pulp/settings.py'"
+            "DJANGO_SETTINGS_MODULE='pulpcore.app.settings' dynaconf list",
```

Comment:
Does this not need a space between the env vars? This will concatenate as `PULP_SETTINGS='/etc/pulp/settings.py'DJANGO_SETTINGS_MODULE='pulpcore.app.settings'` currently.

Reply:
Good eyes! Thanks, fixed now.